int.go

```diff
@@ -274,6 +274,8 @@ func (z *intLogger) logJson(t time.Time, level Level, msg string, args ...interf
 		}
 	}
 
+	args = combineArgs(z.implied, args)
```

Comment:
Given we copy the slice in `With` to make sure it's independent of the parent Logger, we don't need to copy it again here right before using it. While it might be safer to do so, it also puts a pretty heavy additional GC operation in the fast path for logging, which I believe we can avoid.

Reply:
This comment would apply to `log` then too

Reply:
Yes, copying the slice here was solely because of lingering concerns about impact to `z.implied`, but I did not come up with a problematic path.  I'll replace them with the original `append`  and move combineArgs code into `With`.